src/RouteCollection.php

```diff
@@ -127,6 +127,12 @@ private function injectGroup(Group $group, array &$tree, string $prefix = ''): v
 
             /** @var Route $modifiedItem */
             $modifiedItem = $item->pattern($prefix . $item->getPattern());
+            if ($group->getHost() !== null && $modifiedItem->getHost() === null) {
+                $modifiedItem = $modifiedItem->host($group->getHost());
+            }
+            if ($group->getName() !== null && strpos($modifiedItem->getName(), implode(', ', $modifiedItem->getMethods())) === false) {
```

Comment:
What is the mean of checking `strpos($modifiedItem->getName(), implode(', ', $modifiedItem->getMethods())) === false` ?

Reply:
Checking that the route has a default name. We skip unnamed routes.

Reply:
May be create method `$route->isDefaultName(): bool` ?

Reply:
> May be create method $route->isDefaultName(): bool ?

What do you guys think @yiisoft/yii3?

Reply:
Seems to be alright.